src/main/java/frc/robot/subsystems/SwerveDrive.java

```diff
@@ -127,6 +130,20 @@ public void resetOdometry(Pose2d newPose) {
         m_odometry.resetPosition(newPose.getRotation(), DRIVETRAIN.getModulePositions(), newPose);
     }
 
+    public Command resetAtReef(Vision vision) {
+        return Commands.runOnce(() -> {
+            Pose2d newPose = Constants.PathplanningConstants.aprilTagPoseMap.get(vision.april(true));
```

Comment:
tady je potřeba udělat offset, takhle by nám to posadilo střed robota přímo na AprilTag, což nechceme - sem bych pastnul to odhadování pozice robota, který budeme dělat na začátku autonomního